Adyen/Core/Payment Methods/Abstract/PaymentMethodType.swift

```diff
@@ -37,7 +37,17 @@ public enum PaymentMethodType: RawRepresentable, Hashable, Codable {
     case econtextStores
     case econtextATM
     case econtextOnline
+    case boletoBancario
+    @available(
+        *,
+        deprecated,
+        renamed: "boletoBancarioSantander",
+        message: "Use 'boletoBancarioSantander' instead. Consider 'boletoBancario' as the default payment method for Boleto Bancario."
```

Comment:
The message text is a bit confusing starting from the `Consider...` part. As in, if boletoBancario is default why is it renamed type of confusion.

Reply:
There are two issues here:
1. What was named as `boleto` before, in reality represents the tx-variant "boletoBancarioSantander". To prevent confusion and mismatch we decided to deprecate and rename into `boletoBancarioSantander`.
2. Since we were only offering `boletoBancarioSantander`, merchants had one option only so that's always their default tx-variant. However, from the business side, merchants are always encouraged to use `boletoBancario` since its success rate is higher. (this information came from @Robert-SD who's in touch with IMs working with Boleto)